[T9][F09-C4]

rickyyx opened the pull request.

daredevil999 committed:
Morphed Model.Person to Model.Task
Removed the need for tags and uniqueTagList

daredevil999 committed:
Morphed the main Model package.

rickyyx committed:
Complete documentation for v0.0 (#5)

* update docs for v0.0

* minor changes

* update images

rickyyx committed:
Enable add floating tasks without UI updates

rickyyx committed:
Enable GUI add floating task commands

rickyyx committed:
Add pull request format to developer guide

daredevil999 committed:
Merge pull request #8 from CS2103AUG2016-F09-C4/morph_add_floatingtask

Morph-Enable GUI adding floating task command

daredevil999 committed:
Morph of test package to enable building of project (#11)

* Morphed the test package.
No more compilation errors in test package

Yeehengg committed:
Mulitple files morph from addressbook to taskbook

Yeehengg committed:
image file removed due to not relevent to current project

Yeehengg committed:
-Replaced AddressBookStorage with TaskBookStorage
-Replaced XmlAdaptedPerson with XmlAdaptedTask

-Renamed XmkSerializableAddressBook to XmkSerializableTaskBook
-Renamed XmkAddressBookStorage to XmkTaskBookStorage

Yeehengg committed:
Modify and added multiple files to morph addressbook to taskbook

Yeehengg committed:
-AboutUs.md added back

Yeehengg committed:
Added files that are modified after morphing addressBook to taskBook

Yeehengg committed:
-edited changes highlighted by @KM
to rename addressbook to taskbook

Yeehengg committed:
Added files that after edited after morphing addressbook to taskbook

Yeehengg committed:
Added files that are edited after morphing addressbook to taskbook

Yeehengg committed:
Edited files in src/test/java/guitests to morph addressbook to taskbook

rickyyx committed:
Morph UI package

rickyyx committed:
Morph storage package

rickyyx committed:
Update docs-adding images

daredevil999 committed:
Edited design architecture diagram

rickyyx committed:
Fix photo out of position on AboutUs.md

rickyyx committed:
Restore to KianMing's version with no compliation error

rickyyx committed:
Build successfully on local machine with non-omitted tests passed

rickyyx committed:
Modify config json file key names

TiankaiChen committed:
Morph Commons Package and MainApp (#18)

daredevil999 committed:
Merge pull request #17 from CS2103AUG2016-F09-C4/succ_build

Success build the projects with non-omitted test passed

TiankaiChen committed:
Integrate Event and Task Configuration

TiankaiChen committed:
Rework Event and Task classes

TiankaiChen committed:
Removed Tag class

TiankaiChen committed:
Changed copy constructor of Event class

daredevil999 committed:
Merge pull request #21 from CS2103AUG2016-F09-C4/Integrate_Event_and_Task_Configuration

Integrate Event and Task Configuration

daredevil999 committed:
Create abstract Parser interface and add command parser (#24)

* Added Regex for Deadline for task and Duration for event

* Added a Parser Interface for all the AddTaskParser, AddEventParser, ... classes
The Switch Case is managed by ParserManager.

daredevil999 committed:
Implement add command for task and event (#31)

* To implement functionalities for add task and add event. All tests created before have passed, indicating addition of tasks to be successful.

* For implementation of Add Event to Model

* Added several tests for Adding of Events

rickyyx committed:
Enable list task command with [all] option

rickyyx committed:
Move COMMAND_USAGE into superclass ListCommand

daredevil999 committed:
Merge pull request #34 from CS2103AUG2016-F09-C4/list_task

Enable list task command with [all] option

daredevil999 committed:
Implemented an event list display panel (#35)

TiankaiChen committed:
Implement delete command for task and event (#37)

* Add Delete task and event commands

* Fix regex issue

* Fix Test files

* Update .gitignore to solve gradle.prefs issue

* Revised .gitignore

* Rework abstract class DeleteCommand and Pass all tests

* Revised flags for Delete command

* Fix DeleteCommandTest

* Fix minor mistakes in comments

rickyyx committed:
Enable flexible duration with prettydate library (#43)

* Enable flexible duration with prettydate library

* Update gradle dependencies to include external jar

* Include external jar in push

* Update build.gradle

TiankaiChen committed:
Implement mark command to task (#44)

* Add Mark Command

* Add MarkParser

* Add JUnit test cases for Mark Command

* Brought back SelectCommand for use in the future

* Fix Parser Manager to handle select command

* Fix select test case in logic manager test

rickyyx committed:
Include dependencies in classpath

daredevil999 committed:
Updated UML Diagrams

daredevil999 committed:
Edited UserGuide

daredevil999 committed:
Edited UserGuide

daredevil999 committed:
Edited User Guide

daredevil999 committed:
Edited User Guide

rickyyx committed:
Enable List event with -a flags (#48)

* Enable flexible duration with prettydate library

* Update gradle dependencies to include external jar

* Include external jar in push

* Update build.gradle

* Enable list event with [-a] flag with logic tests passed

* Enable list event to pass GUITests

* Fix Errors

* Fix dependencies

* Resolve conflict which omitted in last commit

daredevil999 committed:
Created an abstract AddCommand class (#50)

* Created an abstract AddCommand class

* Edited messages in add parser for incorrect command

* Edited test case

rickyyx committed:
Implement find for both events and tasks in both names and description (#51)

daredevil999 committed:
Update DeveloperGuide according to tutor feedback

daredevil999 committed:
Improve add command (#45)

* Created an abstract AddCommand class

* Enable add floating task and task with deadline functionality

* Deadline can now be shown on Task Card

* Work In Progress:To enable saving of task and event and to pass all test cases

* Implement floating tasks and Improve on EventDuration format

rickyyx committed:
Update Travis on master

rickyyx committed:
Make logic test OOP

daredevil999 committed:
Merge pull request #60 from CS2103AUG2016-F09-C4/morph_logic_test

Make logic test OOP

Yeehengg committed:
Edited and standardlize all message status in each commands (#63)

daredevil999 committed:
Implement edit task (#61)

* enabled edit command for tasks

* Added tests for Edit Command Logic

* Added GUI tests for Edit Task Command

daredevil999 committed:
Update Edit Command userguide

rickyyx committed:
Improve Deadline class

rickyyx committed:
Improve event duration

daredevil999 committed:
Merge pull request #67 from CS2103AUG2016-F09-C4/improve_ddl_format

Improve Deadline format

TiankaiChen committed:
Fix mark command (#66)

* Fixed Mark Command and Junit Test cases

* Add assertions

* delete redundant code

* Switched to OOP

* Remove LogicManagerTest cotent

daredevil999 committed:
Improve edit task command (#64)

* Edited edit command implementation with comments from TK

* Minor edits for exisiting tests for Edit command

* Added a test to check for a duplicate edited task against an exisitng task

* Corrected test case parsed index

rickyyx committed:
Update DeveloperGuide for list, find, undo

rickyyx committed:
Fastforward master

TiankaiChen committed:
Implement clear command (#73)

* Implement Clear Command

* Fix Parser

* Fix issue with clear of completed task and events

* Fix Mark command

* Fix parser issue

* Fix Merge conflict with master

* Add Clear Command test cases.

* Fix FindCommandTest case with modified clear command

* Add Clear Command test heuristics and fix minor errors

* Add author's name to relevant commands, parsers and Junit tests
Add test heuristics for delete and mark command
Add utility function in Commandtest
Fix test case for delete command

* Fix Message_Usage for clear command

* Modified Clear Command's execution, removed null case return

TiankaiChen committed:
Update diagrams and userguide

daredevil999 committed:
Implement edit event (#76)

* Implement edit event command

* Fixed minor bug to do with duration

* Enabled editing of deadline task

* Added tests in EP fashion

* Edited according to TK's comments

TiankaiChen committed:
Modify changes to UserGuide

daredevil999 committed:
Update UserGuide

TiankaiChen committed:
Implement clear command (#81)

* Implement Clear Command

* Fix Parser

* Fix issue with clear of completed task and events

* Fix Mark command

* Fix parser issue

* Fix Merge conflict with master

* Add Clear Command test cases.

* Fix FindCommandTest case with modified clear command

* Add Clear Command test heuristics and fix minor errors

* Add author's name to relevant commands, parsers and Junit tests
Add test heuristics for delete and mark command
Add utility function in Commandtest
Fix test case for delete command

* Fix Message_Usage for clear command

* Modified Clear Command's execution, removed null case return

* Modify regex of clear command

* Resolve problems

rickyyx committed:
add slack integration to travis

rickyyx committed:
update docs

rickyyx committed:
include coverage

Yeehengg committed:
Reupload pull request on Implement_Help_Command (#74)

* Implemented Help Function

* no message

* Updated message status for help command
Updated Help parser

* -Edit clear command message status.
-Transferred clear message usuage from help command to clear command

* -Edited message usuage for exit command
- transferred exit message usuage from help command to exit comannd

* updated the change made in help command in LogicManagerTest

* no message

* Reupload pull request on Implement_Help_Command

* Edited and removed errors in codes

* Delete LogicManagerTest.java

* Edited HelpManagerTest

* Fixed bugs and build errors

* Added heuristic test for help command

* Fixing the error found by travis

* ignore helpcommandtest

Yeehengg committed:
Select Command (#75)

* Work-In-Progess
- Not able to work
- Request to review codes

* Added SelectParser
Added SelectTaskCommand
Added SelectEventCommand

Added JumpToEListRequestEvent
Rename JumpToListRequestEvent to JumpToTListRequestEvent

Refactor affected file with JumpToListRequestEvent to JumpToTListRequestEvent

* Edited files for select command to work

* Refactor JumpToEListRequestEvent to JumpToEventListRequestEvent
Refactor JumpToTListRequestEvent to JumpToTaskListRequestEvent

Added affected files after refactoring

* Removed typo error and commented codes

rickyyx committed:
Implement Undo for tasks and events add & delete (#68)

* Implement undo logic for add, delete

* Enable undo for task

* Enable event undo

* Add gui tests for undo, implement undo for mark

* Improve on exception message for undo command

* move UndoableCommandHistory

* task listed sorted by first deadline then name

* implement sorting of events and tasks by time and then name

* Refactor undo

* implement undo for edit, clear

rickyyx committed:
update gradle for shadowJar

rickyyx committed:
add codacy

chanjunweimy commented on docs/DeveloperGuide.md:
```diff
@@ -70,16 +71,16 @@ Each of the four components
 
 >Note how the `Model` simply raises a `ModelChangedEvent` when the model is changed,
  instead of asking the `Storage` to save the updates to the hard disk.
 
```
Model doesn't store address book data now :P

chanjunweimy commented on docs/DeveloperGuide.md:
```diff
@@ -243,54 +244,250 @@ a. Include those libraries in the repo (this bloats the repo size)<br>
 b. Require developers to download those libraries manually (this creates extra work for developers)<br>
```
Good job in describing the architectures... However, there are a few other things you can include such as you can explain how your design support undo/redo. There are multiple ways this can be done, so it will be useful to explain how you do it and why you chose to do it that way.

chanjunweimy replied:
Similarly, you can explain other interesting design decisions you made.

chanjunweimy replied:
It may be more practical to give a few examples of interesting test cases and explain the thinking behind how you arrived at those test cases, rather than try to give a comprehensive list of all test cases.

chanjunweimy commented on docs/UserGuide.md:
```diff
@@ -11,117 +12,273 @@
 **A**: Install the app in the other computer and overwrite the empty data file it creates with 
-       the file that contains the data of your preious Address Book.
+       the file that contains the data of your previous Task Book.
        
```
In future, you might want to use partial screenshots to show GUI changes for some commands... For example, if user types in a wrong command, it will show a red box, and a feedback message. :)

chanjunweimy commented on docs/UserGuide.md:
```diff
@@ -11,117 +12,273 @@
+[Find](#finding-for-events/tasks) | `find -t|-e KEYWORD [MORE_KEYWORDS]`
+[Clear Tasks or Events](#clearing-completed/uncompleted-tasks/events) |`clear -t|-e [-a]`
+[Clear Tasks and Events](#clearing-completed/uncompleted-tasks and events) |`clear [-a]`
+[Exit](#exiting-the-program) | `exit`
```
The 'Command summary' section will be used heavily by our testers. Try to make it easy to use as possible. e.g. give examples for each command.

chanjunweimy reviewed (commented):
@githubxxcc Good job for this week! You all are ahead of track! Keep up the good work ^^

daredevil999 committed:
Make add input for task flexible (#82)

* Changed description to an optional field for adding a task

* Ensured all existing tests have passed

* Added more test case to AddCommand for tasks in EP fashion

* Edited comments for EP

* Make argument input order flexible for adding tasks

* Added argument tokenizer test

* Added a test case to test for varying ordered inputs

* Edited according to codacy

* Deleted regex and made use of ArgumentTokenizer

* Added more symbols to be accepted in description and name

* Redesigned add task command constructor

* Redesigned task constructor

* Removed duplicate catch statements

* Edited to getDescritptionValue() in model manager

Yeehengg committed:
Edited help command and help parser to reflect undo command (#98)

* - Edited help command and help parser to reflect undo command
- Edited exit command message usage

* fix bugs

rickyyx committed:
Improve EditEvent and AddEvent related functionalities (#96)

* modify event duration

* Refined edit event

* Ensured all tests have passed

* Ensured passing of tests

* Added edit event command tests

* Added AddEvent gui tests

* Added edit event GUI tests

* remove debugging

* Made event description optional

* Solved bugs

* Added "-" to list of accepted symbols

Yeehengg committed:
Added Help Command Test (#97)

* Added Help Command Test

* - Edited help command and help parser to reflect undo command
- Edited exit command message usage

* fix bugs

* fix bugs

* Fix another one in AddCommandTest

* -Fixed HelpCommandTest error
-Edit HelpCommad and Undo Command message_usage formatting

* Edited method name for codacy

daredevil999 committed:
Update UserGuide.md

For add and edit. Change all instances of TaskBook to dowat.

daredevil999 committed:
Updated UserGuide.md

Missed some spots

Yeehengg committed:
-Changed prefix from -t, -e > /t, /e (#104)

-> Select Command and parser
-> Help Command and Paser

- Edited test cases to the new prefix

- Added author

daredevil999 committed:
Updated AboutUs.md

According to v0.4 requirements format.
TODO: Link to our respective codes.md and to key in our major contributions

daredevil999 committed:
Updated our profile link in AboutUs.md

TiankaiChen committed:
Implement misc things (#102)

* Implement offline viewing of User Guide.

* Changed flags for delete and clear command

* Change flags and modified the date format in userguide

* Add extreme case EP to clear command test

* changed flags for deletecommandtest

* Fixed flags in FindCommandTest and UndoCommandTest

daredevil999 committed:
Updated AboutUs.md

Added KM's major contributions and hyperlinks

rickyyx committed:
Improve UI  (#105)

* modify event duration

* Refined edit event

* Ensured all tests have passed

* Ensured passing of tests

* Added edit event command tests

* Added AddEvent gui tests

* Added edit event GUI tests

* remove debugging

* Made event description optional

* Solved bugs

* Added "-" to list of accepted symbols

* Show calendar without data

* Enable refresh of events when added

* Implementing changing calendar time span

* Implemented change of time frame and switch of mode

* Enable highlight when selected

* Show all events at init

* add more guitests for calendar

* Complete GUI test for calendar

* Add panel title and update mock ui

Yeehengg committed:
Update AboutUs.md (#108)

rickyyx committed:
add about XC

TiankaiChen committed:
Alter flags and tagging (#112)

* alter flags

* Done tagging for my parts

daredevil999 committed:
Set background colour and text for completed tasks and events (#111)

TiankaiChen committed:
Improve test cases (#109)

* Add GUI test for Clear Command

* Add GUI test for Delete Command

* Add GUI Test for Mark Command

* Fixed test cases

* Tagged my test cases

TiankaiChen committed:
Improve command box (#106)

* Implement CommandBox modifications

* Fix and implement improvements to commandbox

* Add GUI test for CommandBox

* done tagging for commandbox improvements

rickyyx committed:
Improve on test cases and documentation  (#110)

* modify event duration

* Refined edit event

* Ensured all tests have passed

* Ensured passing of tests

* Added edit event command tests

* Added AddEvent gui tests

* Added edit event GUI tests

* remove debugging

* Made event description optional

* Solved bugs

* Added "-" to list of accepted symbols

* Show calendar without data

* Enable refresh of events when added

* Implementing changing calendar time span

* Implemented change of time frame and switch of mode

* Enable highlight when selected

* Show all events at init

* add more guitests for calendar

* Complete GUI test for calendar

* change argtokenizer

* Improve flags, test cases, userguide

* add authors

* more documentations

* update docs

daredevil999 committed:
Added Keith's @author

daredevil999 committed:
Added km @author for docs

daredevil999 committed:
Add KM @author for main

daredevil999 committed:
Collated main, docs and test for everyone

daredevil999 committed:
Merge pull request #114 from CS2103AUG2016-F09-C4/Add_@author

Add @author

daredevil999 committed:
updated aboutus.md with collated files

rickyyx committed:
v0.4

TiankaiChen committed:
Update Userguide

TiankaiChen committed:
Update Userguide html

rickyyx committed:
Update docs

daredevil999 committed:
Improve UI Display Task and Event Card (#118)

* Improved UI for task and event card

* Minor edits to @@author for KM

* Increased default window size and height

* Solved minor bug

* Resolved bugs with tests

* Trimmed extra space in event desciption to string

* Solved bugs

* Improving Test Coverage percentage

* Implemented Select command tests

* To improve test coverage

rickyyx committed:
merge with KM

rickyyx committed:
change UI theme

rickyyx committed:
add icon and color

rickyyx committed:
refine UI

rickyyx committed:
add task into calendar

rickyyx committed:
fixing build

rickyyx committed:
fix another bux on testtask

rickyyx committed:
delete recurrence

TiankaiChen committed:
Implement focus traversal (#124)

* change UI theme

* add icon and color

* refine UI

* add task into calendar

* fixing build

*  fix another bux on testtask

* Implement Tab event filter

* Fix CommandBox indexOutOfBounds

* Change focus color and faint focus color

* Add safeguards and Add Mouse selection

* Fix minor bugs

* Add Main Window Test cases

rickyyx committed:
solve conflict

daredevil999 committed:
Added activity diagrams

daredevil999 committed:
Edited UI class diagram

daredevil999 committed:
Edited minor alighment error

daredevil999 committed:
Edited User Guide

daredevil999 committed:
Merge branch 'Add_activity_diagram' of https://github.com/CS2103AUG2016-F09-C4/Main into Add_activity_diagram

daredevil999 committed:
Merge pull request #123 from CS2103AUG2016-F09-C4/refine_UI

Refine UI

daredevil999 committed:
Changed naming of image

daredevil999 committed:
Edited DeveloperGuide with AD

daredevil999 committed:
Edited developer guide

include AD

daredevil999 committed:
Changed AD image

daredevil999 committed:
Updated developerguide.md

daredevil999 committed:
Merge pull request #127 from CS2103AUG2016-F09-C4/Add_activity_diagram

Add activity diagram

TiankaiChen committed:
Amend Userguide to include UI Controls

rickyyx committed:
Implement Flex find (#126)

* flexible find with word distance less than 3

* update test cases and userdoc, and add power search mode

TiankaiChen committed:
Modify documentation

TiankaiChen committed:
Merge branch 'master' of https://github.com/CS2103AUG2016-F09-C4/Main

TiankaiChen committed:
Fix MockUI picture

TiankaiChen committed:
Fix MockUI picture

TiankaiChen committed:
no message

TiankaiChen committed:
Improve coverage and Codacy Standards (#125)

* Add ExitCommandTest case and more test cases

* Add Test cases for CollectionUtil

* Modify CollectionUtilTest to reflect actual usage

* Delete unused code

* Improve coverage for UrlUtilTest

* Improve coverage for UnmodifiableObservableListTest

* Remove redundant code in delete parser

* Increase coverage for UnmodifiableObservableListTest

* Improve Coverage for Undo Command for editing task and event

* Remove all unused imports, improve codacy grade

* add dependencies

* fix the stupid import...

* fix merge error

daredevil999 committed:
Edit task to remove deadline (#128)

* Implemented removing deadline for task by using remove flag

* Added an edit test and improved code quality

* Removed an ignored test

* Added exception handling when removing deadline in floating task

Yeehengg committed:
Implement save command, edit help command and added authors to files (#107)

* Added mutiple files for save command

* Added save command test

* added save command in help parser

* Revert "added save command in help parser"

This reverts commit 79b94705d3dbbe76e3311105abba78a8a3ec075b.

* Added save command to help command and help parser

* Added @@author to files

* Attempt to solve codacy error

* attempt to resolve Travis error for @@author

* - Added all the files, changed made for save commands.
- Mostly similar to  #93298a9,  #c97804b,  #c397771 but made slight changes.
- Travis failed bacause of #df90a6f this commit which is not related to save command.
- Junit all pass for save command and test case

* Edited file to resolve travis error

* Attempt to fix travis error

* attempt to solve travis error - 2
- removed invalid file path check and moved it to logic check

* fix config error

* add SaveCommand class depen

rickyyx committed:
update docs and author

rickyyx committed:
update ui

rickyyx committed:
fix BUG

daredevil999 committed:
Collated files

rickyyx committed:
remove commented codes

TiankaiChen committed:
Amend userguide

TiankaiChen committed:
update userguide.html

TiankaiChen committed:
Update AboutUs

daredevil999 committed:
To implement auto select feature for add edit and mark (#137)

rickyyx committed:
Improve UI and fix display issues  (#136)

* color differnt types of task

* reformat the description

* put floating tasks at the bottom

* remove desc

* change done tasks color

* improve codacy

* Minor edits to the help message for add and edit command

* Edited getAsText() method for better formatting

Yeehengg committed:
added @@author

Yeehengg committed:
edited userguide for select, save and help

Yeehengg committed:
-added select use case
- inserted all the missing <br>

Yeehengg committed:
updated existing collated files and added new collated files

Yeehengg committed:
edited @@author typo

Yeehengg committed:
Update collated file

Yeehengg committed:
Updating help command with show function (#140)

Yeehengg committed:
New Select command test (#139)

* Added Select Command Test

Signed-off-by: Poh Yee Heng <a0125534@u.nus.edu>

* - edited select parser to validate input equals to zero
- edited @@author

Signed-off-by: Poh Yee Heng <a0125534@u.nus.edu>

* edited typo error

Signed-off-by: Poh Yee Heng <a0125534@u.nus.edu>

* To improve Codacy

Signed-off-by: Poh Yee Heng <a0125534@u.nus.edu>

* Revert "To improve Codacy"

This reverts commit 39e3065794645cf5bf09569508522091e3af6148.

rickyyx committed:
change argstokenizer to remove excessive exception && add CalendarView enum

rickyyx committed:
update UI part of userguide && refine List command

rickyyx committed:
Add today event alert

rickyyx committed:
remove redundacncy in model

rickyyx committed:
fix the index not showing problem when name too long

rickyyx committed:
update guipref

daredevil999 committed:
Improve code quality for KM part (#146)

* Improve code quality for code

* Allowing ' symbol for name and description field

* Edited AddCommandTest for code quality

* Improved editcommandtest code quality

daredevil999 committed:
Update Readme.md with Dowat title1

rickyyx committed:
reduce redundant codes

daredevil999 committed:
Merge pull request #145 from CS2103AUG2016-F09-C4/improve_SLAP

Improve slap and update user guide for ui

daredevil999 committed:
Improve dev guide (#147)

* Updated general errors in Developer Guide

* Edited diagrams according to TK comments

TiankaiChen committed:
Fix minor issues (#148)

* Update Logging

* Update documentation to userguide

* Fix documentation of userguide

* Update Developer Guide use cases for clear command and UI controls

* Fix minor errors after merge conflict resolution

rickyyx committed:
Improve on DeveloperGuide (#149)

* update logic diagram and list SD

* add constribution

* update UI

* fix some missing parts after merge

rickyyx committed:
fix formatting for use cases

rickyyx committed:
update gradle to change App title

rickyyx committed:
update typo in packages and improve developer guide

rickyyx committed:
add gif

daredevil999 committed:
Refine dev user guide (KM Part) (#151)

* edited dev guide

* edited user guide

* updated userguide

* updated deve guide

daredevil999 committed:
Merge pull request #153 from CS2103AUG2016-F09-C4/last_sprint

Add gif and update package naming

Yeehengg committed:
Documentation (#150)

* Edited @@author to be above header comments

* Edited @@author to be above header comments for Test files

* Edited files, changed the names to "Dowat"

* Updated DeveloperGuide for my part

* Edited to solve travis error

* Update .DeveloperGuide.md.html

Edited changes by XC

* edit file, changed naming for log file

* remove html

TiankaiChen committed:
Tiankai improve code quality (#152)

* Improve Code Quality for commands

* Remove redundant code

* Improve code quality for Parser classes

* Improve code quality of UI

* Improve code quality of guitests

* Improve code quality for logic test cases

* Improve code quality for model test cases

* Change author to position above header

* Improve code quality, fix naming conventions

* Remove unused import

* fix import errors

rickyyx committed:
fix badges

daredevil999 committed:
TestScript for manual testing

daredevil999 committed:
Updated dev guide

Merge conflict errors

rickyyx committed:
change status name and Dowat

TiankaiChen committed:
Update userguide with gifs

TiankaiChen committed:
Merge pull request #155 from CS2103AUG2016-F09-C4/Tiankai_update_documentation

Update userguide with gifs

rickyyx committed:
Update DeveloperGuide.md

TiankaiChen committed:
fix formatting

TiankaiChen committed:
Merge branch 'master' of https://github.com/CS2103AUG2016-F09-C4/Main

TiankaiChen committed:
fix userguide

rickyyx committed:
update logic diagram

daredevil999 committed:
Edited TestScript

daredevil999 committed:
Merge branch 'master' of https://github.com/CS2103AUG2016-F09-C4/Main

TiankaiChen committed:
update developer guide

TiankaiChen committed:
Merge pull request #156 from CS2103AUG2016-F09-C4/tk_developer_guide

update developer guide

daredevil999 committed:
Update test script

daredevil999 committed:
updated aboutus

daredevil999 committed:
Upload SampleData

daredevil999 committed:
Updated developer guide

TiankaiChen committed:
update developer guide

TiankaiChen committed:
Merge branch 'master' of https://github.com/CS2103AUG2016-F09-C4/Main

TiankaiChen committed:
update developer guide

TiankaiChen committed:
update dev

TiankaiChen committed:
correct typos

rickyyx committed:
update logic diagram

TiankaiChen committed:
correct typos

TiankaiChen committed:
Merge branch 'master' of https://github.com/CS2103AUG2016-F09-C4/Main

TiankaiChen committed:
Update AboutUs.md

TiankaiChen committed:
Update AboutUs.md

daredevil999 committed:
Update testscript

daredevil999 committed:
update usrguide

daredevil999 committed:
update user guide

daredevil999 committed:
Update add comand in user guide

Yeehengg committed:
Update AboutUs.md

daredevil999 committed:
Updated list in user guide

TiankaiChen committed:
Update DeveloperGuide.md

daredevil999 committed:
Update developer guide

daredevil999 committed:
Update deve guide

TiankaiChen committed:
Fix HelpWindow (#157)

TiankaiChen committed:
Update DeveloperGuide.md

rickyyx committed:
Revert "Fix HelpWindow" (#158)

Yeehengg committed:
Update DeveloperGuide.md

added missing diagrams for select and help

daredevil999 committed:
update user guide

Yeehengg committed:
Update DeveloperGuide.md

Yeehengg committed:
Add files via upload

daredevil999 committed:
update dev guide

Yeehengg committed:
Delete SDSelect1.png

Yeehengg committed:
Add files via upload

rickyyx committed:
fix help window

rickyyx committed:
fix help

rickyyx committed:
collated

rickyyx committed:
add coll

TiankaiChen committed:
Revert helpwindow to addressbook level 4

rickyyx committed:
add messages

Yeehengg committed:
Delete A125534L.md

rickyyx committed:
tag

rickyyx committed:
delete docs

rickyyx committed:
fix Help window popup html

rickyyx committed:
Update README.md

Yeehengg committed:
no message

Yeehengg committed:
no message

Yeehengg committed:
Delete Yee Heng.md

Yeehengg committed:
Delete Yee Heng.md

Yeehengg committed:
Delete A0125534L.md

Yeehengg committed:
Delete A0125534L.md

Yeehengg committed:
Edited Author

Yeehengg committed:
collated

daredevil999 committed:
update user guide

daredevil999 committed:
update user guide
